tests/manage/monitoring/conftest.py

```diff
@@ -143,3 +143,75 @@ def stop_mgr():
     logger.info(f"Upscaling deployment {mgr} back to 1")
     oc.exec_oc_cmd(f"scale --replicas=1 deployment/{mgr}")
     return measured_op
+
+
+@pytest.fixture(scope="session")
+def workload_stop_ceph_mon():
```

Comment:
should this fixture be started before any test? or should we start it only on demand?

Reply:
- I believe that it should be started only on demand by test case. The fixture is of `session` scope so it is started only once per test run. Idea behind this fixture is that it is used by related alerting and monitoring test cases. It contains collected alerts and start and stop time of stopping monitors so it is possible to check metrics from Prometheus API filtered by time (although alerts API doesn't have option to filter alerts by time, so they are collected in another thread during the time that monitors are down in test case from this PR).
- Yes, the fixture is accessible only under `tests/manage/monitoring`.

Reply:
@fbalak thanks for the explanation